README.md

```diff
@@ -855,38 +902,83 @@ no-op for the modules that already deployed successfully, and should only affect
 time around.
 
 
-
-
-
-
-
 ## Terragrunt details
 
 This section contains detailed documentation for the following aspects of Terragrunt:
 
-1. [Helper functions](#helper-functions)
+1. [AWS credentials](#aws-credentials)
+1. [AWS IAM policies](#aws-iam-policies)
+1. [Interpolation Syntax](#interpolation-syntax)
 1. [CLI options](#cli-options)
+1. [Configuration](#configuration)
 1. [Migrating from Terragrunt v0.11.x and Terraform 0.8.x and older](#migrating-from-terragrunt-v011x-and-terraform-08x-and-older)
-1. [Terragrunt config files](#terragrunt-config-files)
 1. [Developing Terragrunt](#developing-terragrunt)
 1. [License](#license)
 
 
-### Helper functions
+### AWS credentials
+
+Terragrunt uses the official [AWS SDK for Go](https://aws.amazon.com/sdk-for-go/), which
+means that it will automatically load credentials using the 
+[AWS standard approach](https://aws.amazon.com/blogs/security/a-new-and-standardized-way-to-manage-credentials-in-the-aws-sdks/). If you need help configuring your credentials, please refer to the [Terraform docs](https://www.terraform.io/docs/providers/aws/#authentication).
+
+
+### AWS IAM policies
+
+Your AWS user must have an [IAM 
+policy](http://docs.aws.amazon.com/amazondynamodb/latest/developerguide/access-control-identity-based.html) 
+which grants permissions for interacting with DynamoDB and S3. Terragrnt will automatically create
+the configured DynamoDB tables and S3 buckets for storing remote state if they do not already exist.
+
+The following is an example IAM policy which the Terragrunt organization would use. The policy grants 
```

Comment:
s/The following is an example IAM policy which the Terragrunt organization would use/+The following is an example IAM policy for use with Terragrunt/